### PR TITLE
Hotfix/link layout

### LINKED ISSUE #83: Stack the homepage learn more section better
Current rendering needs to not crop image and space buttons:
Clipboard 2019-05-11 at 6 55 24 PM

Design mockup:
image

### PR DESCRIPTION
**Pull request by abea**
Closes #83 

I went ahead and applied the same to others on the home page. Also some eslint fixes, though I tried to not go too nuts with that at the risk of muddying the PR.

### CODE REVIEW

File: components/HomePage/Hero.js [hunk 1]
```diff
@@ -39,8 +36,9 @@ const ActionButtons = styled.div`
     padding-bottom: 3rem;
   `};
 
-  ${below.med`
+  ${below.small`
     flex-direction: column;
+    align-items: stretch;
```
Comment (kburnell) [inline components/HomePage/Hero.js]: Love this...would like to get it built in to the base LinkButton component if possible

File: components/HomePage/LearnMore.js [hunk 2]
```diff
@@ -87,4 +91,11 @@ const LearnMore = ({ className }) => {
   );
 };
 
+LearnMore.propTypes = {
```
Comment (kburnell) [inline components/HomePage/LearnMore.js]: In a branch to be merged shortly, still pending review, I have disabled prop validation in eslint.

File: components/HomePage/SpeakerHighlight.js [hunk 3]
```diff
@@ -22,15 +21,25 @@ const ProfileRow = styled.div`
...
 const Header = styled.h3`
   width: 50vw;
+
+  ${below.small`
+    width: auto;
+  `};
+`;
+
+const ActionButton = styled(LinkButton)`
+  margin: 0 1rem;
```
Comment (kburnell) [inline components/HomePage/SpeakerHighlight.js]: 👍 for using rem!

File: components/shared/LinkButton.js [hunk 4]
```diff
@@ -12,6 +14,11 @@ const OutlineLink = styled.a`
   background-color: ${({ backgroundColor, theme }) =>
     backgroundColor ? theme.colors[backgroundColor] : theme.colors.white};
 
+  ${below.small`
+    &.stretch-sm {
```
Comment (kburnell) [inline components/shared/LinkButton.js]: Thinking we should remove the class selector and just make this the default behavior.

File: components/HomePage/LearnMore.js (hunk 2 above)
Comment (abea) [reply components/HomePage/LearnMore.js]: Sounds good. Once that's completed I'd be happy to help clean up eslint warnings and add that to `npm test`.

File: components/HomePage/Hero.js (hunk 1 above)
Comment (abea) [reply components/HomePage/Hero.js]: So this is the wrapper of the LinkButtons components, not the LinkButton itself. A few other places use a similar pattern, but I think we can avoid it completely by removing the linkButton wrapping div. I'm going to give that a shot.

File: components/shared/LinkButton.js (hunk 4 above)
Comment (abea) [reply components/shared/LinkButton.js]: That should work.

File: components/HomePage/SponsorHighlight.js [hunk 5]
```diff
@@ -99,9 +102,10 @@ const SponsorHighlight = ({ className, eventSlug }) => {
               We partner with industry leading organizations and companies that
               are looking to connect with software developers and practitioners.
             </p>
-            <ActionButton
```
Comment (abea) [inline components/HomePage/SponsorHighlight.js]: Renaming is optional, but it seemed like "ActionButton" was being used several places without consistent meaning.

File: components/HomePage/SpeakerHighlight.js [hunk 6]
```diff
@@ -40,14 +51,12 @@ const SpeakerHighlight = ({ className }) => {
         <Header>
           Find Inspiration From Hallway Conversations or Industry Leaders
         </Header>
-        <div style={{ display: 'flex' }}>
```
Comment (abea) [inline components/HomePage/SpeakerHighlight.js]: Less wrapping overall allowed some more consistency with less markup.

**Comment by csell5** [comment]
#### 📝Changed routes:


| [**/wi/become-a-partner**](https://that-website-git-fork-abea-hotfix-link-layout.that-conference.now.sh/wi/become-a-partner) |
|:-:|
|<a href="https://that-website-git-fork-abea-hotfix-link-layout.that-conference.now.sh/wi/become-a-partner">Screenshot of /wi/become-a-partner</a><br /><sup><a href="https://images.weserv.nl?url=microlink.nyc3.digitaloceanspaces.com/_z1gbnejs1UUtdy6A1PlpCutDdQ_">(view full size)</a> |







Commit cc8f5e5bb3012359f70801957c535e91891dd639 (https://that-website-git-fork-abea-hotfix-link-layout.that-conference.now.sh).

**Pull request merged**